backend/src/pubsub.ts

```diff
@@ -59,7 +59,13 @@ export const initPubSub = (
     for (const [name, endpoint] of handlers) {
       const uri = wrapProcedure(config.networkName, name as ProcedureType);
       try {
-        await session.register(uri, (args: any) => endpoint(args, state));
+        await session.register(uri, (args, kwargs) => {
```

Comment:
A weird part of autobahn - we can only use arrays in `args` and object in `kwargs`.
As usage of `cursor` introduced objects in input - I decided to split input depending or being an array both on incoming and outcoming code sections.